api/v1alpha1/cluster_types.go

```diff
@@ -22,8 +22,10 @@ import (
 )
 
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// Important: Run "make" to regenerate code after modifying this file
```

Comment:
Maybe note the specific make task/target?

Reply:
It is an automatically generated comment from kubebuilder.  I have a note to catch this in CI, which I think is the best solution.  I think I will pull the comment.